### DISCUSSION TITLE
Code contributions from outside

### DISCUSSION THREAD

**Discussion by davidmarcelbaum** [emavgl/oinkoin#54] [General]
Hey @emavgl ,

Are you willing to accept contributions from external people (like me) :-) ?

There are some stuff I can help with (without changing the flow and looks of the app) such as widget overflows on small screens.

**Comment by davidmarcelbaum** [comment]
Ah nevermind, I did check the Readme before but I must have missed the section(s!) where you explicitly stated that **contributions are welcome**... Leaving discussion thread open but feel free to close it.

**Comment by emavgl** [comment]
Hi @davidmarcelbaum,

Code contribution are more than welcome! 

If you want to contribute, feel free to open a new PR and add a description on what you are going to fix/develop.

From the PR we can have a discussion about the implementations etc.

When you finish with that, I'll review the change and merge the PR.

I'm closing the issue, but you can still comment if you need.

Thanks again :)
